setup/azurecmdfiles.wxi

```diff
@@ -7554,4 +7554,4 @@
             <ComponentRef Id="cmpE6E248F4029BF7DACCD9C87C2D38BD38" />
         </ComponentGroup>
     </Fragment>
-</Include>
+</Include>
```

Comment:
Revert any change here - this is likely causing your build issues.  Please check out the latest version of the wxi file.